Automattic/pocket-casts-ios
# Transcripts fix support for alternative format types

| 📘 Part of: #2185 |  
|:---:|

Fixes # 

This PR updates the code so transcript format can have more than one valid mime type that match them.

## To test

1. Start the app
2. Open the following podcast: https://pca.st/episode/2cb385e0-e8f2-4b5d-be50-69b203630174
3. Play it and open the full screen player
4. Tap on the Transcript icon
5. Check if the transcript shows correctly


## Checklist
